docs/page.md

```diff
@@ -107,15 +107,15 @@ Example (_Catalog/Page/StorefrontCategoryPage.xml_ file):
 This example shows the page with the name `StorefrontCategoryPage`.
 It will be merged with the other `StorefrontCategoryPage` pages from other modules.
 
-The following is an example of a call in test:
+The following is an example of a call in an action group:
```

Comment:
Any reason to change `test` to `action group`?

Reply:
May be irrelevant here. Fixed.